index.js

```diff
@@ -6,13 +6,14 @@ const render = require('./app/render.js');
 
 const OUTPUT_FILE_NAME = 'index.html';
 
-const html = render(content);
+const hash = Date.now();
+const html = render('./components/index.njk', Object.assign(content, { hash }));
 
 fs.writeFile(
-  path.join(__dirname, 'build', OUTPUT_FILE_NAME),
+  path.join(__dirname, 'dist', OUTPUT_FILE_NAME),
   html,
   err => {
     if (err)  throw err;
-    else      console.log(`Generated ${OUTPUT_FILE_NAME}.`);
+    else      process.stdout.write(hash.toString());
```

Comment:
yeah – it's so I can use the same hash in the HTML builder and webpack. It's not an ideal solution, but seems like the quickest and dependency-less one.